### PR TITLE
docs(readme): update slack community link

### PR DESCRIPTION
**Pull request by lplazas**
### Checklist prior to review

- [x] [Link to relevant issue][1] as well as docs and UI issues --
**Not creating to avoid extra toil**
- [x] This will not break child repos: it doesn't hardcode values (.e.g "kumahq" as a image registry) and it will work on Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
- [x] Tests (Unit test, E2E tests, manual test on universal and k8s) --
  - Don't forget `ci/` labels to run additional/fewer tests
- [x] Do you need to update [`UPGRADE.md`](../blob/master/UPGRADE.md)? --
- [x] Does it need to be backported according to the [backporting policy](../blob/master/CONTRIBUTING.md#backporting)? ([this](https://github.com/kumahq/kuma/actions/workflows/auto-backport.yaml) GH action will add "backport" label based on these [file globs](https://github.com/kumahq/kuma/blob/master/.github/workflows/auto-backport.yaml#L6), if you want to prevent it from adding the "backport" label use [no-backport-autolabel](https://github.com/kumahq/kuma/blob/master/.github/workflows/auto-backport.yaml#L8) label) --

[1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**None of the above needed as it is just a documentation change**

### CODE REVIEW

**Review by lahabana — APPROVED** [review]

**Comment by lahabana** [comment]
Thx a lot!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ contributors and maintainers.
 
 [Slack](https://kuma-mesh.slack.com) is the main chat channel used by the
 community and the maintainers of this project. If you do not have an
-existing account, please follow this [link](https://join.slack.com/t/kuma-mesh/shared_invite/zt-1rcll3y6t-DkV_CAItZUoy0IvCwQ~jlQ) to sign
+existing account, please follow this [link](https://join.slack.com/t/kuma-mesh/shared_invite/zt-1tu98qcw4-pJNW9lRdBqx1I9kF9rvImA) to sign
```
Comment (slonka) [inline CONTRIBUTING.md:13]: No idea why the old one expired (I did not do anything with my account, do they expire base on time?). @lahabana - should we bring back the invite bot somehow?

Comment (slonka) [reply CONTRIBUTING.md:13]: Also this link was present in multiple places so need to update all of them 😢

Comment (slonka) [reply CONTRIBUTING.md:13]: Ok, now that I re-generated the link it said that it expires in 30 days. I'm 99% sure that was not the case last time I generated it.

image

We probably don't want to repeat this every month so probably need to invest in something like https://communityinviter.com/ ?

**Pull request merged**